.github/workflows/verify_source_urls.yml

```diff
@@ -5,4 +5,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - run: python3 ./tools/verify_stable_archives.py
\ No newline at end of file
+      - name: Setup Bazel
+        uses: abhinavsingh/setup-bazel@v3
```

Comment:
there's no need for this actually, all GitHub actions worker images (`ubuntu-latest` here) already have bazelisk installed.

Reply:
Good to know! I just based it on https://github.com/bazel-contrib/bazel-mypy-integration/blob/abd703906b901b6b5841b59b47bc0301ffa5d8ac/.github/workflows/continuous-integration.yml